roles/lib_utils/library/delegated_serial_command.py

```diff
@@ -192,9 +192,9 @@ def main():
             fcntl.flock(lockfd, fcntl.LOCK_EX | fcntl.LOCK_NB)
             break
         # pylint: disable=invalid-name
-        except IOError as e:
-            if e.errno != errno.EAGAIN:
-                module.fail_json(msg="I/O Error {0}: {1}".format(e.errno, e.strerror))
+        except IOError as ioerror:
```

Comment:
Is it just a cosmetic change or some bugfix?

Reply:
Yes, this was reporting a flake8 or pylint issue.